action.js

```diff
@@ -20,18 +20,23 @@ function loadFile(location) {
         reject(error);
       });
     };
-    if (location.startsWith('http://')) {
-      http.get(location, httpCallback);
-    } else if (location.startsWith('https://')) {
-      https.get(location, httpCallback);
-    } else {
+    if (fs.existsSync(location)) {
       fs.readFile(location, 'utf8', (err, data) => {
         if (err) {
           reject(err);
         } else {
           resolve(data);
         }
       });
+    } else {
+      let url = new URL(location);
+      if (url.protocol === 'http:') {
```

Comment:
do we need to support http anyway? :)

Reply:
Well, we could remove it to enforce HTTPS usage, but I am worried there may be some schemas that are only available through HTTP. But I don't have strong opinion here.